sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_service_client_helper.py

```diff
@@ -0,0 +1,28 @@
+# -------------------------------------------------------------------------
```

Comment:
Theoretically should be `_data_lake_service_client_helpers.py` with an `s`.

Reply:
Not just theoretical :)

Reply:
I also don't love the idea of having just one function in the helpers. I didn't see any other need to factor out functionality in the service client, and same with file system client as well. Should we get rid of the helpers file and say it's okay for some code duplication?